js/changeColor.js

```diff
@@ -0,0 +1,63 @@
+import {storage} from './main.js'
+
+let btn = document.getElementById('change-color-btn');
+
+/* Need this since the color picker wants hex 
+** to set the default value and apparently js
+** just convert hex to rgb when I put in a hex
+** value into the backgroundColor
+*/ 
+function RGBToHex(rgb) {
```

Comment:
I think we should stay consistent with the arrow functions

Reply:
hehe I totally did not just copy and paste this :P